app/screens/search/search.js

```diff
@@ -701,17 +702,21 @@ export default class Search extends PureComponent {
             }
             break;
         case RequestStatus.FAILURE:
-            results = [{
-                id: RequestStatus.FAILURE,
-                component: (
-                    <View style={style.searching}>
-                        <PostListRetry
-                            retry={this.retry}
-                            theme={theme}
-                        />
-                    </View>
-                ),
-            }];
+            if (postIds.length) {
+                results = postIds;
+            } else {
```

Comment:
If requests fail with having some previous posts then there is not retry available?

Reply:
the retry will happen when continuing to scroll down, if we don't do this the whole result list will be replaced with a retry button removing all the previous loaded posts